x-pack/plugins/security_solution/public/detections/components/rules/pre_packaged_rules/update_callout.tsx

```diff
@@ -14,19 +14,46 @@ import * as i18n from './translations';
 interface UpdatePrePackagedRulesCallOutProps {
   loading: boolean;
   numberOfUpdatedRules: number;
+  numberOfUpdatedTimelines: number;
   updateRules: () => void;
 }
 
 const UpdatePrePackagedRulesCallOutComponent: React.FC<UpdatePrePackagedRulesCallOutProps> = ({
   loading,
   numberOfUpdatedRules,
+  numberOfUpdatedTimelines,
   updateRules,
 }) => {
   const { services } = useKibana();
+
+  const prepackagedRulesOrTimelines = useMemo(() => {
+    if (numberOfUpdatedRules > 0 && numberOfUpdatedTimelines === 0) {
```

Comment:
It might be possible that we need to show both messages to the users

Reply:
Make sense, but in that case the button title would be long. If we have messages from both rules and timelines, Line 40 covers that case, what do you think?